playbooks/roles/airship-setup-deployer/tasks/main.yml

```diff
@@ -86,17 +92,9 @@
       {{ socok8s_ext_vip }} novncproxy.openstack.svc.cluster.local
       {{ socok8s_ext_vip }} orchestration.openstack.svc.cluster.local
 
-- name: Creates /etc/openstack directory
-  become: yes
-  file:
-    path: "{{ item }}"
-    state: directory
-    #   owner: "{{ ansible_user }}"
-  with_items:
-    - /etc/openstack
-
-- name: Copy clouds.yml
-  become: yes
+- name: Save default clouds.yaml file
   template:
-    src: clouds.yml.j2
-    dest: /etc/openstack/clouds.yaml
+    src: clouds.yaml.j2
+    dest: /root/.config/openstack/clouds.yaml
```

Comment:
why moving to /root?

Reply:
I'm actually not sure now why did I do that change originally (there's a link to the original commit).  I remember @dirkmueller suggested it and it sounded reasonable

Reply:
does that mean now we require only run as root user?

Reply:
That would require it, yes. Actually you might be right that keeping it under /etc/ looks better